bucket/kops.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.11.0",
+    "description": "Kubernetes Operations (kops) - Production Grade K8s Installation, Upgrades, and Management.",
+    "homepage": "https://github.com/kubernetes/kops/",
+    "license": "Apache License 2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/kubernetes/kops/releases/download/1.11.0/kops-windows-amd64#/kops.exe",
+            "hash": "996f1607a226f5459cbf590d7e6a5ee7f04ffe1ae757ae37d01a23ee7ceb24a2"
+        }
+    },
+    "bin": "kops.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/kubernetes/kops/releases/download/$version/kops-windows-amd64#/kops.exe"
+            }
+        }
```

Comment:
Add hash extraction

```suggestion
        },
        "hash": {
            "url": "$url-sha1"
        }
```

Reply:
Run checkver with Force parameter afterward

Reply:
Go to root of repository and execute `.\bin\checkver.ps1 kops -f`

Reply:
Done.

Reply:
Done.

Reply:
What's the reason you haven't used `$url` variable?? 

![Proof](https://i.imgur.com/BnMPBd2.png)